src/components/Card.js

```diff
@@ -12,34 +16,58 @@ export default class Card {
       .content
       .querySelector('.card')
       .cloneNode(true);
-
     return cardElement;
   }
 
+  _renderLikes() {
+    if (this._likeCountElement) {
+      this._likeCountElement.textContent = this._likes.length;
+    }
+  }
+
+  _setEventListeners() {
+    this._cardImage.addEventListener('click', () => {
+      this._handleCardClick(this._name, this._link);
+    });
+
+    this._likeButton.addEventListener('click', () => {
+      const isLiked = this._likeButton.classList.contains('card__like-button_active');
+      this._likeButton.classList.toggle('card__like-button_active'); 
+      localStorage.setItem(this._id, JSON.stringify(this._likeButton.classList.contains('card__like-button_active')));
```

Comment:
`localStorage` can't be used for **likes handling**. You should check the card data. You have `isLiked` inside it. This is what is needed to deside whether to show `black` or `white` heart